CMakeLists.txt

```diff
@@ -1,6 +1,43 @@
 cmake_minimum_required(VERSION 2.8)
 project(succinct CXX)
 
+FIND_PACKAGE(PythonInterp "3.7")
+
+if (PYTHONINTERP_FOUND)
+  if (UNIX AND NOT APPLE)
+    if (PYTHON_VERSION_MAJOR EQUAL 3)
+        FIND_PACKAGE(Boost COMPONENTS python${PYTHON_VERSION_SUFFIX})
```

Comment:
Code style: Stick to lower-case for cmake functions, e.g., find_package(...), enable_testing(), include_directories(...), link_libraries(...), etc.